### PR TITLE
require user, allow group override

### PR DESCRIPTION
**Pull request by TheKevJames**

### CODE REVIEW

File: tests/init.pp [hunk 1]
```diff
@@ -1 +1,4 @@
-class { 'homebrew': }
+class { 'homebrew':
+  user  => 'root',
```
Comment (amosshapira) [inline tests/init.pp]: Does this work?

Comment (TheKevJames) [inline tests/init.pp]: It works on Travis, yes, but often CI systems do some weird things to make root-only access work. I have not tested whether the root user is useable on a standard system, but Homebrew itself advises installing/using it as a non-root user.

**Comment by jordigg** [comment]
I wouldn't let the user select user and/or group and instead move everything to be under the root user and the admin group: http://apple.stackexchange.com/questions/42127/homebrew-permissions-multiple-users-needing-to-brew-update

Something I do too is to set the default set up path to be /Applications rather than the /Users/user/Applications so everything that gets installed is available on the Applications folder of all users and is always located on the same applications folder `/Applications/`. I do that by setting an environment variable called `HOMEBREW_CASK_OPTS --appdir=/Applications`. Maybe we could allow the parameter `--appdir` when installing packages. Like `brew cask install --appdir="~/Applications" google-chrome`

**Comment by TheKevJames** [comment]
Interesting idea. Moved it to #24 for further consideration.

**Pull request merged**